runtime/doc/treesitter.txt

```diff
@@ -1218,9 +1218,9 @@ LanguageTree:trees({self})                              *LanguageTree:trees()*
 
 
 ==============================================================================
-Lua module: vim.treesitter.playground              *lua-treesitter-playground*
```

Comment:
We should not include this module at all in our documentation at the moment (if at all); the only function `vim.treesitter.dev.inspect_tree` is exported and more fully documented as `vim.treesitter.inspect_tree`.